BUILD.md

```diff
@@ -0,0 +1,140 @@
+# Build Issues and Solutions
+
+## Prerequisites
+
+Check that you have **DashSync** repo next to the wallet repo.
+
+Make sure you're on the **master** branch for both DashSync and DashWallet.
+
+### Clang Version Check
+
+Check clang version:
+
+```bash
+clang++ --version
+```
+
+If you see homebrew version, this might cause issues with building. You need to switch to system version.
+
+```
+❌ InstalledDir: /opt/homebrew/opt/llvm/bin
+✅ InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
+```
+
+To unplug homebrew version, comment out the following in `~/.zshrc` or `~/.zsh_profile` (or `~/.bashrc` if you're using bash):
+
+```bash
+export PATH="/opt/homebrew/opt/llvm/bin:$PATH"
+```
+
+## Required Dependencies
+
+Install the following dependencies:
+
+- **cmake** (version 3.28.3 recommended)
+- **cbindgen**
+- **rust**
+
+> **Note:** cmake higher than 3.5.0 might cause issues with building. Version 3.28.3 is tested and works.
```

Comment:
The messages when we had cmake 4.0 installed:  CMake < 3.5 is not supported by the relic library.  Not sure why this would be the case if the version was 4.  The recording was checked for this.

Perhaps we should say that CMake > 4.0 might cause issues with building.

Reply:
Yeah, maybe